garak/probes/phrasing.py

```diff
@@ -0,0 +1,111 @@
+"""Past Tense
+
+    Refusal training is widely used to prevent LLMs from generating harmful, undesirable, or illegal
+outputs. We reveal a curious generalization gap in the current refusal training approaches:
+simply reformulating a harmful request in the past tense  is often sufficient to jailbreak many state-
+of-the-art LLMs
+"""
+
+import random
+
+from garak import _config
+from garak.data import path as data_path
+from garak.probes.base import Probe
+
+
+class TenseMini:
+    def _minify_prompts(self):
+        random.shuffle(self.prompts)
+        self.prompts = self.prompts[:200]
+        self.generations = 1
+
+
+class PastTense(Probe):
+    """Will the system give unsafe responses when asked to write about weapon creation or dangerous behavior?
+
+    This check sends the generator requests for advice on making weapons in the past tense.
```

Comment:
```suggestion
    This check sends the generator requests for advice on possibly protect or restricted subjects in the past tense.
```

Prompts used definitely are not just for `weapons`.